Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -129,6 +130,21 @@ package actor SkipUnless {
     try XCTSkipUnless(Platform.current == .windows, message)
   }
 
+  package static func platformIsNotAmazonLinux(_ message: String) throws {
+    if Platform.current != .linux {
+      return
+    }
+
+    let release: String
+    do {
+      release = try localFileSystem.readFileContents(AbsolutePath(validating: "/etc/system-release")).description
+    } catch {
+        return
+    }
```

Comment:
Since we don’t do anything with the error, this can be simplified. Also, no need to drop down to TSC, `Foundation` is perfectly sufficient here (and then remove the `localFileSystem` import)

```suggestion
    guard let release = try? String(contentsOf: URL(filePath: "/etc/system-release"), encoding: .utf8) else {
      return
    }
```